packaging/ubuntu-16.04/snapd.postrm

```diff
@@ -137,7 +137,7 @@ if [ "$1" = "purge" ]; then
     echo "Final directory cleanup"
     for d in "/snap/bin" "/snap" "/var/snap"; do
         # Force remove due to directories for old revisions could still exist
-        rm -rf "$d"
+        rm -rf "$d" || true
```

Comment:
At this point all snaps should be stopped and  unmounted - or are we hit here by the fact that we cannot reliably stop user services yet?

Reply:
or maybe due to this: https://github.com/snapcore/snapd/pull/9522 ?